security-monkey/templates/config-map.yaml

```diff
@@ -21,6 +21,9 @@ data:
   SECURITY_MONKEY_EMAIL_DEFAULT_SENDER: "{{ .Values.smConfig.adminTeamEmail }}"
   SECURITY_MONKEY_SMTP: "{{ .Values.smConfig.emailUseSMTP }}"
   SECURITY_MONKEY_SES_REGION: "{{ .Values.smConfig.awsSesRegion }}"
+  SECURITY_MONKEY_EMAIL_USERNAME: "{{ .Values.smConfig.emailSMTPUsername }}"
+  SECURITY_MONKEY_EMAIL_PASSWORD: "{{ .Values.smConfig.emailSMTPPassword }}"
```

Comment:
Shouldn't this be `emailSMTPUser` ?

Reply:
It's just to be consistent with the env variable wording. (why User and not Pass then?)
Not big deal :)